ground/src/main/java/com/google/android/ground/util/StreamUtil.kt

```diff
@@ -22,26 +22,11 @@ import timber.log.Timber
 
 /** Methods for working with and manipulating [java8.util.stream.Stream]. */
 object StreamUtil {
```

Comment:
We should also rename this class to something more appropriate since `Stream` isn't used anymore

Reply:
What would be a better name? Hopefully when moving to coroutines this helper won't be needing anymore.

Reply:
The remaining method is only used in 1 class. We can even consider removing this util and making this a private method.

Reply:
Good catch. Fixed!